config/default/common/config/metadata/layers/seawifs/SEAWIFS_ORBVIEW-2_GAC_True_Color.md

```diff
@@ -0,0 +1,5 @@
+The SeaWiFS Corrected Reflectance (True Color) layer provides true-color or natural color images because the combination of wavelengths produces and image that is similar to what the human eye would see. The images are natural-looking images of land surface, oceanic and atmospheric features.
```

Comment:
Typo here: "...combination of wavelengths produces and image that...". "and" should be "an".